Create ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md (#2573)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-0-0-change-label-of-the-position-input-in-the-dashboard-settings-7240.md

---
date: ""

> title: Change label of the position input in the dashboard settings (#7240)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Rename "Position in navigator" field to "Position in tabs" in dashboard settings
```

> title: Rename "Position in navigator" field to "Position in tabs" in dashboard settings

Reviewer: ```suggestion
title: Renamed "Position in navigator" field in the dashboard settings
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60191
version: 1021.0.0
---

> Change label of the position input in the dashboard settings (#7240)

Author: The Claude AI suggested to use the following body:
```suggestion
The "Position in navigator" field in dashboard settings is now labeled as "Position in tabs" for better clarity.
```

> The "Position in navigator" field in dashboard settings is now labeled as "Position in tabs" for better clarity.

Reviewer: ```suggestion
For better clarity, the **Position in navigator** field in the dashboard settings has been renamed to **Position in tabs**.
```
